src/wrappers/themis/jsthemis/index.d.ts

```diff
@@ -0,0 +1,60 @@
+export declare class KeyPair {
```

Comment:
did you write this file manually or somehow generated from the sources?

Reply:
Manually. I did not find the tool that can read `.node` file to make a declaration file.